CHANGELOG.md

```diff
@@ -1,2 +1,3 @@
 - Replaces `tslint` with `eslint` as the default linter in new Cloud Functions for Firebase directories. (#1663)
 - Fixes bug where scheduled Cloud Functions for Firebase deploys unexpectedly succeeded with invalid Cloud Scheduler configurations. (#2609)
+- Add support for Firebase Remote Config (see README for list of commands)
```

Comment:
Judging based on past changelog entries, I think this should probably say something along the lines of:

```suggestion
- Adds Firebase Remote Config commands: `remoteconfig:get`, `remoteconfig:versions:list`, `remoteconfig:rollback`.
- Adds a `remoteconfig` target for `firebase deploy`.
```

Reply:
Thanks for the suggestion! Updated :)